### ISSUE TITLE
Improve submissions form accessibility

### ISSUE THREAD

**Issue by yshyonli2** [shlmagazine/website#18]
- [ ] Make radio buttons focusable (perhaps by resizing the radio button and adding a fill-in-the-blank bubble on the left)
- [ ] Change background color to white, which changes to black when selected (similar to checkbox style)
- [ ] Style spinner (and maybe add some “please wait” text)
